docs/getting-started/advanced-setting.asciidoc

```diff
@@ -76,6 +76,7 @@ IMPORTANT: Threat intelligence indices aren't required to be ECS-compatible for
 
 Kibana transmits certain information about Elastic Security when users interact with the {security-app}, detailed below. {kib} redacts or obfuscates personal data (IP addresses, host names, usernames, etc.) before transmitting messages to Elastic. Security-specific telemetry events include:
 
+* *Detection rule security alerts:* Information about Elastic-authored prebuilt detection rules using the detection engine. Examples of alert data include machine learning job influencers, process names, and cloud audit events.
```

Comment:
```suggestion
* *Detection rule security alerts:* Information about Elastic-authored prebuilt detection rules using the detection engine. Examples of alert data include machine learning job influencers, process names, and Elastic Cloud audit events.
```

Reply:
Suggesting this edit in case you're describing the Elastic Cloud and not the cloud in general. Here are our guidelines for using the term in case you'd like more info:
https://wiki.elastic.co/display/MAR/Writing+resources+for+Elasticians#WritingresourcesforElasticians-ElasticCloud

Thanks for writing this @pjhampton!

Reply:
Thanks, @nastasha-solomon! Actually, it's not Elastic Cloud, it is public cloud audit events such as [Cloud Trail](https://aws.amazon.com/cloudtrail/). We of course strip away any sensitive information. How do you advise we proceed if `cloud` is too ambiguous?